stdlib/ysh/args-test.ysh

```diff
@@ -347,6 +347,41 @@ proc test-multi-value-flags {
   assert [2 === _error.code]
 }
 
+proc test-escaped-args {
+  parser (&spec) {
+    flag -f --float (Float)
+    flag -i --int (Int)
+    arg pos
+    rest rest
+  }
+
+  var args = parseArgs(spec, :|-f 1.0 pos_arg -- -i foo bar|)
+
+  assert [type(args.float) === 'Float']
```

Comment:
I know this file is inconsistent, but we should have `assert ['Float' === `, with expected on the left

Similar to `assert [2 === `

Because the error message actually does say "expected" and "got".  (This order is conventional in Python and Java I believe)